README.md

```diff
@@ -28,7 +28,7 @@ Usage
 Only two params are required:
```

Comment:
We should update this to `only one param is required`

Reply:
Resolved in [1f9614a](https://github.com/elboletaire/manga-downloader/pull/7/commits/1f9614ad550860dd411a87a46e1f824f928d0569)